src/pages/commits/InactiveCommits/index.tsx

```diff
@@ -0,0 +1,133 @@
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import Tooltip from "@leafygreen-ui/tooltip";
+import { Body } from "@leafygreen-ui/typography";
+import { string } from "utils";
+
+const { getDateCopy } = string;
+const { gray } = uiColors;
+
+type rolledUpVersion = {
+  id: string;
+  author: string;
+  createTime: string;
+  order: number;
+  message: string;
+  githash: string;
+};
+interface InactiveCommitsProps {
+  rolledUpVersions: rolledUpVersion[];
+}
+export const InactiveCommits: React.FC<InactiveCommitsProps> = ({
+  rolledUpVersions,
+}) => {
+  const versionCount = rolledUpVersions.length;
+
+  const shouldSplitCommits = versionCount > MAX_COMMIT_COUNT;
+
+  let returnedCommits = [];
+  if (shouldSplitCommits) {
+    const hiddenCommitCount = versionCount - MAX_COMMIT_COUNT;
+    returnedCommits = rolledUpVersions.slice(0, 2).map((v) => (
+      <CommitText key={v.githash} data-cy="commit-text">
+        {getCommitCopy(v)}
+      </CommitText>
+    ));
+    returnedCommits.push(
+      <HiddenCommitsWrapper
+        key="hidden_commits"
+        data-cy="hidden-commits"
+      >{`${hiddenCommitCount} more commit${
+        hiddenCommitCount !== 1 ? "s" : ""
+      }`}</HiddenCommitsWrapper>
+    );
+    returnedCommits = [
+      ...returnedCommits,
+      ...rolledUpVersions.slice(-3).map((v) => (
+        <CommitText key={v.githash} data-cy="commit-text">
+          {getCommitCopy(v)}
+        </CommitText>
+      )),
+    ];
+  } else {
+    returnedCommits = rolledUpVersions.map((v) => (
+      <CommitText key={v.githash} data-cy="commit-text">
+        {getCommitCopy(v)}
+      </CommitText>
+    ));
+  }
+
+  return (
+    <Tooltip
+      align="top"
```

Comment:
Based on the Figma and playing around with the Tooltip component [on mongodb.design](https://www.mongodb.design/component/tooltip/example/), I think align should be set to `bottom`